mantidproject/mantid
# Redirect help buttons on MuonAnalysis to rst docs

In PR #17118, the documentation for MuonAnalysis was moved to an rst file.
This PR updates links to it:
- Link at http://www.mantidproject.org/Muon changed
- Wiki pages redirected to new documentation (not deleted in case anybody has bookmarked them)
- Help buttons ("?") in the MuonAnalysis interface now open the new help instead of the wiki.

The only code changes here are to which pages the help buttons bring up. Previously they opened a browser at the relevant wiki page, now they open the Mantid help system.

**To test:**
- Go to http://www.mantidproject.org/Muon, the "Interface overview" for Muon Analysis should point to the new documentation.
- Visit the old wiki pages below and check they all link to the new documentation:
  - http://www.mantidproject.org/MuonAnalysis
  - http://www.mantidproject.org/MuonAnalysisHome
  - http://www.mantidproject.org/MuonAnalysisGrouping
  - http://www.mantidproject.org/MuonAnalysisDataAnalysis
  - http://www.mantidproject.org/MuonAnalysisResultsTable
  - http://www.mantidproject.org/MuonAnalysisSettings
- Build Mantid and the Qt help pages. In each tab of the Muon Analysis interface, click the help ("?") button and check the correct page is shown.

(Builds available at `\\olympic\babylon5\Scratch\TomPerkins\PR17325`)

Fixes #17119.

Release notes were updated in #17118.

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
